test/config/featureFlags/QueryNotificationMessageConfigSpec.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config.featureFlags
+
+import base.UnitWithMocksSpec
+import com.typesafe.config.{Config, ConfigFactory}
+import play.api.Configuration
+
+class QueryNotificationMessageConfigSpec extends UnitWithMocksSpec {
+
+  private val configWithEnabledQueryNotificationMessage: Config =
+    ConfigFactory.parseString("microservice.services.features.queryNotificationMessage=enabled")
+  private val configWithDisabledQueryNotificationMessage: Config =
+    ConfigFactory.parseString("microservice.services.features.queryNotificationMessage=disabled")
+  private val emptyConfig: Config =
```

Comment:
Nit: I'd move these 3 vals, `configWithEnabledQueryNotificationMessage`, `configWithDisabledQueryNotificationMessage` and `emptyConfig`, to their specific tests... as local vals.